ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,175 @@
+
+  - *Key*  
+   Action key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
+
+  - *Queue type*  
+  Click the drop-down list to select the applicable queue type. For detailed information about queue types, see [Configure the queue types](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md).
+
+  - *Task event*  
+  Click the drop-down list to select the applicable task event.
+
+  > [Info] This option is linked to the *Tasks* module. It is only displayed if the *Tasks* module is installed and the user has the necessary rights. The *ActindoWorkflow Process Execution Failed* task is preconfigured by default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
+
+  - *Configuration*  
+  Depending on the action, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the action configuration requires advanced programming knowledge.
+
+  > [Info] Not all actions have configuration options. In this case, the *Configuration* section is not displayed.
+
+  - *Static inputs*  
+  All unlinked inputs are listed in this section. The static inputs determine the data type that can be input in a port. They are preconfigured but can be further defined by the user clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button. Only valid JSON values are accepted. When being edited, static inputs display a small arc without a place in front of them in the workflow diagram.
+
+[comment]: <> (Unsure! RS Oli/Hannah)
+
+
+## Input and output ports
+
+![Input and output ports](/Assets/Screenshots/ActindoWorkFlow/Workflows/NewPlace.png "[Input and output ports]")
+
+All actions contain a certain, predefined number of input and output ports. They are defined with a descriptive name, such as origin, data or destination.
+
+Input ports are positioned to the left of the action and output ports are positioned to the right of the action. Both are represented by a square.
+
+Both input and output ports can be of two types:
+
+- **Mandatory**  
+  Mandatory ports must be linked to a place, which in turn must be linked to an action. Places linked to a mandatory port cannot be deleted or removed. When adding a new action, mandatory input/output places are already displayed.
+
+- **Optional**  
+ Optional ports are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to each of them. By clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button a new place is added. which can in turn be linked to a new action. For detailed information about adding new actions, see [Create an action](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-an-action).
+
+
```

Comment:
add info about static inputs:
-> Gitlab: Normally, all tokens for the transitions execution are taken from the places connected to its input ports. But sometimes you want to configure static values for certain inputs. This is possible using the *static inputs*: If a transition has configured a static input on one or more of its input ports, it will behave is if there was a place connected to that input port that always contains one token (containing whatever value is statically configured). You still need at least one "actual" place with an "actual" token in order to make that transition executable.